logger.gd

```diff
@@ -631,8 +641,7 @@ func load_config(configfile = default_configfile_path):
 	# Load modules config and initialize them
 	modules = {}
 	for module_cfg in config.get_value(PLUGIN_NAME, "modules"):
-		var module = Module.new(module_cfg["name"], module_cfg["output_level"], \
-				module_cfg["output_strategies"], get_logfile(module_cfg["logfile_path"]))
+		var module = Module.new(module_cfg["name"], module_cfg["output_level"], module_cfg["output_strategies"], get_logfile(module_cfg["logfile_path"]))
```

Comment:
Would it keep things multiline if written this way?
```suggestion
		var module = Module.new(
				module_cfg["name"],
				module_cfg["output_level"],
				module_cfg["output_strategies"],
				get_logfile(module_cfg["logfile_path"])
		)
```
or similar?

Reply:
No, this will be formatted to one line.